build/ps-pre-stop.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+if [ ${CLUSTER_TYPE} == "async" ]; then
+	exit 0
+fi
+
+LOG_FILE=/var/lib/mysql/pre-stop.log
+NAMESPACE=$(</var/run/secrets/kubernetes.io/serviceaccount/namespace)
+OPERATOR_PASSWORD=$(</etc/mysql/mysql-users-secret/operator)
+FQDN="${HOSTNAME}.${SERVICE_NAME}.${NAMESPACE}"
+
+echo "$(date +%Y-%m-%dT%H:%M:%S%Z): Removing ${FQDN} from cluster" >>${LOG_FILE}
+mysqlsh -i -u operator -p${OPERATOR_PASSWORD} -e "dba.getCluster().removeInstance('${FQDN}:3306')" >>${LOG_FILE} 2>&1
```

Comment:
maybe better to use admin port

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/535/commits/caa4e70a1570a13463922b01353b977f490de522